run.go

```diff
@@ -57,6 +57,10 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 			Name:  "no-new-keyring",
 			Usage: "do not create a new session keyring for the container.  This will cause the container to inherit the calling processes session key",
 		},
+		cli.IntFlag{
+			Name:  "preserve-fds",
+			Usage: "Pass N additional file descriptors to the container (stdio + $LISTEN_FDS + N in total)",
```

Comment:
For this i'm not sure defining individual fds is optimal.  If you have a large amount of fds to pass you would run out of space for all the `--preserve-fd 3 --preserve-fd 4 --preserve-fd 5...`